### PR TITLE
feature_34_FAVORITES_VIEW

### PR DESCRIPTION
**Pull request by Symphonie234**
- Added another blade file to show list of favorited listings
- Added new controller and new web route

### CODE REVIEW

File: app/Http/Controllers/FavoriteController.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * Add a listing to the user's favorites.
+     *
+     * @param  \App\Models\Listing  $listing
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function add(Listing $listing)
+    {
+        // Create a new favorite record for the current user and listing
+        Favorite::create([
```
Comment (ChronoDevs) [inline app/Http/Controllers/FavoriteController.php]: make error response if creation is failed

File: app/Http/Controllers/FavoriteController.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+     */
+    public function remove(Listing $listing)
+    {
+        // Find and delete the favorite record for the current user and listing
+        $favorite = Favorite::where('user_id', auth()->id())
+            ->where('listing_id', $listing->id)
+            ->first();
+
+        if ($favorite) {
+            $favorite->delete();
```
Comment (ChronoDevs) [inline app/Http/Controllers/FavoriteController.php]: make error response if deletion is failed

File: app/Http/Controllers/FavoriteController.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        return back()->with('success', 'Listing removed from favorites!');
+    }
+
+    public function addToFavorites(Listing $listing)
```
Comment (ChronoDevs) [inline app/Http/Controllers/FavoriteController.php:76]: add comment method above

**Comment by Symphonie234** [comment]
- Added error responses on both _**add**_ and _**remove**_ methods in the _**FavoriteController**_.
- Added comment above the _**addToFavorites**_ method in the _**FavoriteController**_.

**Pull request merged**